src/lib/chips/chip-list.ts

```diff
@@ -572,8 +572,10 @@ export class MdChipList implements MdFormFieldControl<any>, ControlValueAccessor
     // Defer setting the value in order to avoid the "Expression
     // has changed after it was checked" errors from Angular.
     Promise.resolve().then(() => {
-      this._setSelectionByValue(this.ngControl ? this.ngControl.value : this._value, false);
-      this.stateChanges.next();
+      if (!!this.ngControl || !!this._value) {
```

Comment:
You don't need the `!!` in conditional expressions, only when the value will be passed around somewhere

Reply:
Done. PTAL. Thanks for review.

Reply:
Need to push your changes?

Reply:
Changes are pushed. Thanks!